pkg/stanza/operator/input/windows/xml.go

```diff
@@ -13,6 +13,7 @@ import (
 
 // EventXML is the rendered xml of an event.
 type EventXML struct {
+	Original         string      `xml:"-"`
```

Comment:
Q.: do we want to have the ability to drop this string? I'm thinking that since it is redundant with the other fields (when not using raw) one may wants to optimize and drop it early. Anyway, it would be an optimization.

Reply:
I think this is effectively handled when we convert the event to a log entry, because we either set the body to this field, _or_ we call `formattedBody` which creates a map that does not contain it.

Reply:
Similarly though - should we be able to automatically save the original along with the formatted version? I think we established on the issue that we should have the option, but I figured this PR is complicated enough that it could wait.

Reply:
Agreed: let's handle it in separate PRs.

Reply:
I opened https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/34756 to track it